src/assets/GeoJSON/eco2go_features.hjson

```diff
@@ -1,50 +1,656 @@
+// use google maps for coordinates, more accurate
+
 {
   "type":"FeatureCollection",
   "name": "OSU Eco2Go",
   "features":[
   {
     "type": "Feature",
-    "geometry":{
+    "geometry":
+    {
       "type":"MultiPoint",
       "coordinates": [
```

Comment:
Why did we change this from being a multi-point geometry object to being a bunch of individual objects?  This seems needlessly verbose to me.

Reply:
> Why did we change this from being a multi-point geometry object to being a bunch of individual objects? This seems needlessly verbose to me.

I think Rishabh made it individual objects to label each building. If it's not a desirable change, we can revert to my commit (https://github.com/OSU-Sustainability-Office/sustainability_map/pull/30/commits/8422cc7305f55ec81aa2e3e907aecf534e003fc4), that should have all the building checks but without individual building objects.